src/instructlab/server.py

```diff
@@ -127,7 +127,19 @@ def ensure_server(
         count = 0
         while server_process.is_alive():
             sleep(0.1)
-            if count > 10:
+            try:
+                list_models(
+                    api_base=temp_api_base,
+                    tls_insecure=tls_insecure,
+                    tls_client_cert=tls_client_cert,
+                    tls_client_key=tls_client_key,
+                    tls_client_passwd=tls_client_passwd,
+                )
+                break
+            except ClientException:
+                pass
+            if count > 50:
+                logger.error("failed to start server")
```

Comment:
```suggestion
                logger.error("failed to reach the API server")
```

The server did not fail, the API is just not returning anything. The server not starting is handled by the upper `while` loop.